scripts/default-env.sh

```diff
@@ -0,0 +1,7 @@
+export PROOT=/home/vincent/dev/osIROSE-new
```

Comment:
oups, my bad, should have checked that ... (It's not my fault if bash can't read my mind and not expand the variable when it should but that's not what I wanted to do ! ^^  )